pvt/helpers/src/contract.ts

```diff
@@ -67,7 +65,6 @@ export function getArtifact(contract: string): Artifact {
     artifactsPath = `${packagePath}/artifacts`;
   }
 
-  const artifacts = new Artifacts(artifactsPath);
   return artifacts.readArtifactSync(contract.split('/').slice(-1)[0]);
```

Comment:
So we don't need the `artifactsPath` anymore? how is the global `artifacts` linked to the path in this case, considering we have multiple workspaces in the monorepo?